### DISCUSSION TITLE
All 7.4 Download folders are empty except for header and avatar

### DISCUSSION THREAD

**Discussion by ethan-edgelord** [UltimaHoarder/UltimaScraper#1462] [Q&A]
All content is being skipped except for the profile header and avatar. 

Python looks like:
0
Type: Profile
100%|██████████████████████████████████████████████████████████████████████████████| 1.59M/1.59M [00:07<00:00, 226kB/s]
Type: Stories
No Stories Found.
Type: Posts
Scrape Attempt: 1/100
Found: Posts
Scraping [photo]. Should take less than a minute.Scraping [photo]. Should take less than a minute.

Scraping [video | stream | gif]. Should take less than a minute.
Scraping [video | stream | gif]. Should take less than a minute.Scraping [audio]. Should take less than a minute.

Scraping [audio]. Should take less than a minute.Scraping [text]. Should take less than a minute.

Scraping [text]. Should take less than a minute.
Processing metadata.
Finished processing metadata.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 194e05269f09, content
INFO  [alembic.runtime.migration] Running upgrade 194e05269f09 -> 5b4bea08c27f, content
INFO  [alembic.runtime.migration] Running upgrade 5b4bea08c27f -> 6b1b10eb67de, content
Renaming files.
Type: Archived
Scrape Attempt: 1/100
Found: Archived_posts
Scraping [photo]. Should take less than a minute.
Scraping [video | stream | gif]. Should take less than a minute.
Scraping [audio]. Should take less than a minute.
Scraping [text]. Should take less than a minute.
Processing metadata.
Finished processing metadata.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 194e05269f09, content
INFO  [alembic.runtime.migration] Running upgrade 194e05269f09 -> 5b4bea08c27f, content
INFO  [alembic.runtime.migration] Running upgrade 5b4bea08c27f -> 6b1b10eb67de, content
Renaming files.
Type: Messages
Processing metadata.
Finished processing metadata.
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> 2c36fcc0b921, content
INFO  [alembic.runtime.migration] Running upgrade 2c36fcc0b921 -> d0118d8ec0b4, content
INFO  [alembic.runtime.migration] Running upgrade d0118d8ec0b4 -> aeb9fe314556, content
Renaming files.
Scrape Completed

**Comment by ProfessionalHorder** [comment]
And then 'enter' once more? Then the downloading process will start.
